samples/powerstore_v230_k8s_121.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: storage.dell.com/v1
+kind: CSIPowerStore
+metadata:
+  name: csi-powerstore
+  namespace: csi-powerstore
+spec:
+  driver:
+    # Config version for CSI PowerStore v2.3 driver
+    configVersion: v2.3.0
+    # Controller count
+    replicas: 1
+    dnsPolicy: ClusterFirstWithHostNet
+    forceUpdate: false
+    fsGroupPolicy: ReadWriteOnceWithFSType
+    common:
+      # Image for CSI PowerStore driver v2.3
+      image: "dellemc/csi-powerstore:nightly"
```

Comment:
Please change it to point to v2.3.0. Also please change this in all samples where ever applicable

Reply:
Done